puente-etl/extract_back4app_data.py

```diff
@@ -0,0 +1,76 @@
+from utils.clients import Clients
+from utils.constants import Outputs
+from load_to_s3 import \
+    export_to_s3_as_json, \
+    export_to_s3_as_pickle_dict, \
```

Comment:
@bcrant what does this pickle library do? If you have a link to a resource, I can just read up on it